contracts/azorius/Azorius.sol

```diff
@@ -0,0 +1,463 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@gnosis.pm/zodiac/contracts/core/Module.sol";
+import "./interfaces/IBaseStrategy.sol";
+import "./interfaces/IAzorius.sol";
+
+/// @title Azorius - A Zodiac module that enables a voting agnostic proposal mechanism
+contract Azorius is Module, IAzorius {
+    bytes32 public constant DOMAIN_SEPARATOR_TYPEHASH =
+        0x47e79534a245952e8b16893a336b85a3d9ea9fa8c573f3d803afb92a79469218;
+    bytes32 public constant TRANSACTION_TYPEHASH =
+        0x72e9670a7ee00f5fbf1049b8c38e3f22fab7e9b85029e85cf9412f17fdd5c2ad;
+    uint256 public totalProposalCount; // Total number of submitted proposals
+    uint256 public timelockPeriod; // Delay between a proposal is passed and can be executed
+    address internal constant SENTINEL_STRATEGY = address(0x1);
```

Comment:
Yeah, so this naming was leftover from Usul. This is the same pattern used by Gnosis Safe for managing module addresses: 

https://github.com/safe-global/safe-contracts/blob/main/contracts/base/ModuleManager.sol#L31

https://en.wikipedia.org/wiki/Sentinel_value

Reply:
yeah, we should keep it since it follows their pattern, but some comments like @sethhrbek said is a good call.  I would even link to the wiki for it ha